includes/abstracts/class-wc-gateway-mondido-abstract.php

```diff
@@ -414,6 +414,16 @@ public function updateOrderWithIncomingProducts( $order, array $transaction ) {
 				continue;
 			}
 
+			$product = wc_get_product($product_id);
+
+			if ($product) {
```

Comment:
What if there's no product found? And this was the only thing bought. The customer will then have paid for said item but in the order order confirmation sent from Woo - this product wont be there. Is this expected behaviour or do we wish to alert that stuff went wrong?

Reply:
What will happen is that its going to be added as a fee, that is what happens further down. So the amount will always be correct, but if they removed the product it will be shown as a fee instead. But i will have to add "2 x" in the beginning of the description to show how many was bought, if there was more than one item.

Reply:
i did change it, if it doesnt find the product, just add a dummy one